src/script/repositories/calling/CallingRepository.ts

```diff
@@ -1386,7 +1389,8 @@ export class CallingRepository {
     if (isTelemetryEnabledAtCurrentEnvironment()) {
       this.showCallQualityFeedbackModal(conversationId);
     }
-
+    const serializeSelfUser = this.selfUser ? this.serializeQualifiedId(this.selfUser) : 'unknown';
+    this.logger.info(`Call Epoch Info: _leave, user: ${serializeSelfUser}, conversation: ${conversationId}`);
```

Comment:
Do we need something to identify that it is `MLS 1:1` or `MLS Conference`

Reply:
I also thought about it, but the information is already somewhere in the logs, and actually we are only interested in the group calls.. and the logs themselves will only be temporary (I think). If you insist, I'll change it